src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -1,321 +1,344 @@
+
+from azure.cli.testsdk import (JMESPathCheck)
+
+
+def prepare_containerapp_env_for_app_e2e_tests(test_cls):
+    from azure.cli.core.azclierror import CLIInternalError
+    from .common import TEST_LOCATION
+    rg_name = f'client.env_rg_{TEST_LOCATION}'.lower().replace(" ", "").replace("(", "").replace(")", "")
+    env_name = f'env-{TEST_LOCATION}'.lower().replace(" ", "").replace("(", "").replace(")", "")
+    managed_env = None
+    try:
+        managed_env = test_cls.cmd('containerapp env show -g {} -n {}'.format(rg_name, env_name)).get_output_in_json()
+    except CLIInternalError as e:
+        if e.error_msg.__contains__('ResourceGroupNotFound') or e.error_msg.__contains__('ResourceNotFound'):
+            test_cls.cmd(f'group create -n {rg_name}')
+            test_cls.cmd(f'containerapp env create -g {rg_name} -n {env_name} --logs-destination none')
+            managed_env = test_cls.cmd('containerapp env show -g {} -n {}'.format(rg_name, env_name)).get_output_in_json()
+
+            while managed_env["properties"]["provisioningState"].lower() == "waiting":
+                time.sleep(5)
+                managed_env = test_cls.cmd('containerapp env show -g {} -n {}'.format(rg_name, env_name)).get_output_in_json()
+    return managed_env["id"]
+
+
+def create_containerapp_env(test_cls, env_name, resource_group, location=None):
+    logs_workspace_name = test_cls.create_random_name(prefix='containerapp-env', length=24)
+    logs_workspace_id = test_cls.cmd('monitor log-analytics workspace create -g {} -n {} -l {}'.format(resource_group, logs_workspace_name, TEST_LOCATION)).get_output_in_json()["customerId"]
+    logs_workspace_key = test_cls.cmd('monitor log-analytics workspace get-shared-keys -g {} -n {}'.format(resource_group, logs_workspace_name)).get_output_in_json()["primarySharedKey"]
+
+    if location:
+        test_cls.cmd(f'containerapp env create -g {resource_group} -n {env_name} --logs-workspace-id {logs_workspace_id} --logs-workspace-key {logs_workspace_key} -l {location}')
+    else:
+        test_cls.cmd(f'containerapp env create -g {resource_group} -n {env_name} --logs-workspace-id {logs_workspace_id} --logs-workspace-key {logs_workspace_key}')
+
+    containerapp_env = test_cls.cmd('containerapp env show -g {} -n {}'.format(resource_group, env_name)).get_output_in_json()
+
+    while containerapp_env["properties"]["provisioningState"].lower() == "waiting":
+        time.sleep(5)
+        containerapp_env = test_cls.cmd('containerapp env show -g {} -n {}'.format(resource_group, env_name)).get_output_in_json()
+
+
+def create_and_verify_containerapp_up(
+            test_cls,
+            resource_group,
+            env_name = None,
+            source_path = None,
+            artifact_path = None,
+            image = None,
+            location = None,
+            ingress = None,
+            target_port = None,
+            app_name = None,
+            requires_acr_prerequisite = False):

```

Comment:
Were there are any other changes in this file other than adding `artifact_path` and `requires_acr_prerequisite` to this function? The GitHub diff is off so it's hard to see what else has been changed.

Reply:
Found the issue! It was the Windows line ending messing up with Git. It should show properly now.